dashboard/app/helpers/levels_helper.rb

```diff
@@ -176,7 +176,7 @@ def dsl_defined_options
   # Options hash for Blockly
   def blockly_options
     l = @level
-    throw ArgumentError("#{l} is not a Blockly object") unless l.is_a? Blockly
```

Comment:
this would have actually thrown UndefinedMethod because ArgumentError is a class not a method